CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 4.0.1
+
+[https://github.com/Creuna-Oslo/react-scripts/pull/24]()
+
+- Fixed issue that made `get-configs` crash when user was missing eslint.config file.
```

Comment:
I think it would be good to keep the style of the previous messages (Like `Fixes` instead of `Fixed`)

Reply:
Also, this description seems to be a bit inaccurate. This package doesn't read any eslint config file itself. The problem was that the `get-configs` didn't handle `eslintConfig === null`, right?